sr_camera_pose_estimation/launch/calibration.launch

```diff
@@ -1,33 +1,45 @@
 <launch>
   <arg name="sim" default="false" />
   <arg name="depth" default="false" />
-	<arg name="marker_size" default="9.0" />
-	<arg name="max_new_marker_error" default="0.08" />
-	<arg name="max_track_error" default="0.0" />
-	<arg name="bundle_files" default="$(find sr_description_common)/ar_markers/ar_marker_0_1_2_3_a4.xml" />
+  <arg name="marker_size" default="9.0" />
+  <arg name="max_new_marker_error" default="0.08" />
+  <arg name="max_track_error" default="0.0" />
+  <arg name="bundle_files" default="$(find sr_description_common)/ar_markers/ar_marker_0_1_2_3_a4.xml" />
+  <arg name="continuous" default="false" />
+  <arg name="window_width" default="100" />
+  <arg name="camera_root_frame" default="camera_link" />
 
   <arg if="$(arg depth)" name="cam_image_topic" default="/camera/depth_registered/points" />
   <arg if="$(arg depth)" name="cam_info_topic" default="/camera/depth_registered/camera_info" />	
   <arg if="$(arg depth)" name="output_frame" default="camera_depth_optical_frame" />
   <arg if="$(arg depth)" name="alvar_node" default="findMarkerBundles" />
-  <arg if="$(arg depth)" name="med_filt_size" default=" 10" />
+  <arg if="$(arg depth)" name="med_filt_size" default="10" />
   <arg unless="$(arg depth)" name="cam_image_topic" default="/camera/rgb/image_raw" />
   <arg unless="$(arg depth)" name="cam_info_topic" default="/camera/rgb/camera_info" />	
   <arg unless="$(arg depth)" name="output_frame" default="camera_rgb_optical_frame" />
   <arg unless="$(arg depth)" name="alvar_node" default="findMarkerBundlesNoKinect" />
   <arg unless="$(arg depth)" name="med_filt_size" default="" />
   
-  <node name="ar_track_alvar" pkg="ar_track_alvar" type="$(arg alvar_node)" respawn="false" output="screen" args="$(arg marker_size) $(arg max_new_marker_error) $(arg max_track_error) $(arg cam_image_topic) $(arg cam_info_topic) $(arg output_frame)$(arg med_filt_size) $(arg bundle_files)" />
-
   <node pkg="tf" type="static_transform_publisher" name="ar_bundle_broadcaster" args="0.1 0.58 0 2.35619449019 0 0 ra_base_link ar_marker_bundle_static 1" />
```

Comment:
If they have to adjust that, they should have to think about the frame their adjustments are relative to as well. The camera pose estimator node doesn't care which tf is the marker tf's parent, it just follows the tf tree up until it can join the camera tree with the world tree.